docs/1-introduction/1-3-models/1-3-1-texturing.md

```diff
@@ -0,0 +1,162 @@
+This data will go in a `D3D11_TEXTURE2D_DESC` which "DESC(ribes)" our texture data.
+
+Most images we see in our daily lives will be 24-bit or 32-bit, depending if they have alpha or not. As was noted before, D3D11 does not support 24-bit textures, but what do these values mean?
+The amount of "Bits per Pixel" is what we're talking about when we say an image is N-bit, this commonly translates to "how many channels" a texture has, in order to not overcomplicate things we'll assume all data we're working on is SDR (Standard Dynamic Range) and not HDR (High Dynamic Range)
+The key difference between those two is the range of a color, in SDR colors usually have 8 bits per channel, meaning our colors go from 0 to 255 (0xFF), HDR can take many shapes or forms and has no real consistent format to speak of but values can easily surpass the range of 0 to 1024 or more.
+
+Knowing that we assume SDR formats we'll `switch` on the amount of BPP of common values, anything weird and we're out!
+This is the first spot where we get into texture formats, our first case is "8 bits per pixel", not very common, but easy to handle, for us this means there only exists an "R" channel, no "GBA" to speak of. We'll tell D3D11 the same by specifying `DXGI_FORMAT_R8_UNORM`.
+
+You'll see the enum ends on `_UNORM`, this is 'how' our texture sampler will return the data to us in the shader.
+The following data types exist:
+
+- UNORM		: "Unsigned Normalized", data will be interpreted as a range of 0.0f to 1.0f.
+- SNORM		: "Signed Normalized", data will be interpreted as a range of -1.0f to 1.0f.
+- UINT		: "Unsigned Integer", data will be interpreted as a maximum range of 0 to 4294967295 (0xFFFFFFFF).
+- SINT		: "Signed Integer", data will be interpreted as a maximum range of -2147483648 to 2147483647.
+- FLOAT		: Each channel will be a full floating point value.
+- TYPELESS	: This format is special and can be many things, its up to the shader how to handle this data.
+
+There exists another suffix behind these ones: `SRGB` (`DXGI_FORMAT_R8G8B8A8_UNORM_SRGB`) which will tell the sampler that the data is in SRGB color space.
+
+As we'll assume normal SDR textures with 8 bits per pixel, we end up with either `R8`, `R8G8`, or `R8G8B8A8` depending on how many BPP the image has.
+We'll also use `UNORM` as the math in shaders becomes a lot easier if everything is a neat 0.0f to 1.0f range.
+
+To keep things simple we set ArraySize, MipLevels and SampleDesc.Count to 1, and these will be explained in later chapters.
+After also filling in the Width, Height and Format we're left with two more members `Usage` and `BindFlags`.
+
+`Usage` will tell D3D11 what we're going to do with the texture data, will we adjust it or is it set in stone from the start? 
+This is what we tell it with:
+
+- D3D11_USAGE_DEFAULT	: "We can read and write to this on the GPU" (For textures we want to change on the GPU later (usually through shaders)).
```

Comment:
Table thing here again if possible

Reply:
() inside a () just looks a bit odd here, do we really need the second one?

Reply:
I concur, just remove the inner braces, but keep 'usually through shaders'

Reply:
fixed